corehq/apps/locations/models.py

```diff
@@ -761,3 +766,19 @@ def _unassign_users_from_location(domain, location_id):
             user.unset_location_by_id(domain, location_id, fall_back_to_next=True)
         elif user.is_commcare_user():
             user.unset_location_by_id(location_id, fall_back_to_next=True)
+
+
+def notify_users_at_location(location_id):
+    """
+    Notify users at parent locations that locations fixture has been updated
+    """
+    from corehq.apps.locations.dbaccessors import user_ids_at_locations
+    from corehq.apps.fixtures.models import UserFixtureType
+    from corehq.apps.users.models import CommCareUser
+    from dimagi.utils.couch.database import iter_docs
+
+    user_ids = user_ids_at_locations([location_id])
```

Comment:
This takes a list of location ids, so you could pass in the full list of ancestor location IDs and do it in one query (it should be a relatively small list anyways).